plugins/meson.build

```diff
@@ -13,140 +13,17 @@ plugin_deps = [
   fwupdplugin_rs_dep,
 ]
 
-plugins = {
```

Comment:
I'm not really a fan tbh; what's the logic?

Reply:
Alternative is here: https://github.com/fwupd/fwupd/pull/9008

Reply:
I think the problem with splitting things into little groups is that sometimes the groups overlap, or it gets a bit fuzzy -- e.g. the plugin might be linux specific if it uses a fallback API for instance. We also need to justify what each of the groups actually *are* -- like if ChromeOS is linux or if uefi is os-indep. From a build point of view it's also a lot easier to just say "plugins go here" rather than having to glob. I'll comment on #9008 which like a lot more.